Dockerfile

```diff
@@ -9,7 +10,9 @@ LABEL name="auth-httpd" \
       summary="httpd image with external authentication" \
       description="An httpd image which includes packages and configuration necessary for handling external authentication."
 
-RUN dnf -y --disableplugin=subscription-manager install http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/centos-repos-8.1-1.1911.0.8.el8.x86_64.rpm http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/centos-gpg-keys-8.1-1.1911.0.8.el8.noarch.rpm && \
+RUN dnf -y --disableplugin=subscription-manager install \
+      http://mirror.centos.org/centos/8.2.2004/BaseOS/${ARCH}/os/Packages/centos-repos-8.2-2.2004.0.1.el8.${ARCH}.rpm \
+      http://mirror.centos.org/centos/8.2.2004/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8.2-2.2004.0.1.el8.noarch.rpm && \
```

Comment:
does it make sense to parameterize `8.2-2.2004` and `8.2.2004` to minimize copy/paste errors in the future?

Reply:
Yeah, I thought about it... I can do follow up PRs for all repos, as I basically copied this section from manageiq-base/Dockerfile.